config.go

```diff
@@ -70,6 +92,29 @@ func (a *AuthConfig) validate() error {
 	return nil
 }
 
+// validate validates the LNC configuration.
+func (l *LNC) validate() error {
+	if !l.Enable {
+		return nil
+	}
+
+	switch {
+	case l.Label == "":
+		return errors.New("lnc label required")
+
+	case l.Passphrase == "":
+		return errors.New("lnc passphrase required")
```

Comment:
I added both to the db: words + entropy. Entropy is the main field use in the API but in the config file people need to use words (+ it can be useful in case people want to identify the entry in the db directly)